sdks/python/apache_beam/transforms/util.py

```diff
@@ -223,7 +223,7 @@ def __init__(self,
     if target_batch_duration_secs and target_batch_duration_secs <= 0:
       raise ValueError("target_batch_duration_secs (%s) must be positive" % (
           target_batch_duration_secs))
-    if max(0, target_batch_overhead, target_batch_duration_secs) == 0:
+    if not (target_batch_overhead or target_batch_duration_secs):
```

Comment:
Why did we need this change?

Reply:
In python 3 max(0, None) gives TypeError: unorderable types: NoneType() > int(). The previous two if conditions already check whether the number given is greater than 0. Al that remains to be checked is whether at least one of them is not None.

Reply:
0 is also a falsey value, so isn't checked above, but this check is correct in that case (and arguably more clear as well).